configs/treescriptworker

```diff
@@ -0,0 +1,47 @@
+{
+    "us-east-1": {
+        "type": "treescriptworker",
+        "domain": "srv.releng.use1.mozilla.com",
+        "ami": "ami-18abe670",
+        "subnet_ids": ["subnet-33a98358"],
+        "security_group_ids": ["sg-03e4047e"],
+        "instance_type": "t2.medium",
```

Comment:
Do you know if we need t2.medium, or are you just copying?
This matters especially if we have a 2nd idle instance.

Reply:
This was just copying, but the initial clone of unified is a bit heavy and prone to timeout (locally) -- I could always make puppet pre-seed that on spin-up.

I'm ok with trying something smaller and increasing if we need to.

Reply:
IIRC, t2 can be used in VPC only. Otherwise it should work.